packages/webmachine/webmachine.0.1.0/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "1.2"
+version: "0.1.0"
+maintainer: "Spiros Eliopoulos <spiros@inhabitedtype.com>"
+homepage: "https://github.com/inhabitedtype/ocaml-webmachine"
+dev-repo: "https://github.com/inhabitedtype/ocaml-webmachine.git"
+bug-reports: "https://github.com/inhabitedtype/ocaml-webmachine/issues"
+authors: ["Inhabited Type LLC"]
+license: "PROP"
+
+available: [
+  ocaml-version >= "4.01"
+]
+build: [
+  ["./configure" "--prefix=%{prefix}%"]
+  [make]
+]
+install: [
+  [make "install"]
+]
+remove: [
+  ["ocamlfind" "remove" "webmachine"]
+]
+
+build-test: [
+  ["./configure" "--enable-tests"]
+  [make]
+  [make "test"]
+]
+
+depends: [
+  "ounit" {test}
```

Comment:
Your `_oasis` file says `oUnit (>= 1.0.2)`.  You can use [oasis2opam](https://github.com/ocaml/oasis2opam) to avoid such discrepancies (it will generate the opam files from `_oasis`).